package.json

```diff
@@ -91,7 +91,8 @@
       "/node_modules/",
       "/src/accessibilityIndex.jsx",
       "/src/index.jsx",
-      "/src/components/BackendStatusBanner/"
+      "/src/components/BackendStatusBanner/",
+      "<rootDir>/src/.*/container.jsx"
```

Comment:
Ignore coverage on container components (i.e. connected components, with mapStateToProps and mapDispatchToProps)